### PR TITLE
ddl: Check duplicate when adding column

### PR DESCRIPTION
**Pull request by shenli**
Add a duplicate column is not right.

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: ddl/ddl_test.go [hunk 1]
```diff
@@ -107,8 +107,13 @@ func (ts *testSuite) TestT(c *C) {
 		}
 	}
 	alterStmt = statement("alter table t add column bb int after b").(*stmts.AlterTableStmt)
-	dd.AlterTable(ctx, tbIdent, alterStmt.Specs)
+	err = dd.AlterTable(ctx, tbIdent, alterStmt.Specs)
+	c.Assert(err, IsNil)
 	c.Assert(alterStmt.Specs[0].String(), Not(Equals), "")
+	// Insert a duplicate column will cause error
```
Comment (siddontang) [inline ddl/ddl_test.go]: Inserting a duplicated column

**Comment by shenli** [comment]
@siddontang PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**